esphome/components/online_image/png_image.h

```diff
@@ -25,9 +25,13 @@ class PngDecoder : public ImageDecoder {
   int prepare(size_t download_size) override;
   int HOT decode(uint8_t *buffer, size_t size) override;
 
+  void add_pixels(uint32_t count) { this->pixels_decoded_ += count; }
```

Comment:
to me `decoded_pixels` sounds more idiomatic than `pixels_decoded`, but you are the native speaker :thinking:  
```suggestion
  void add_pixels(uint32_t count) { this->decoded_pixels_ += count; }
```

Reply:
Also, maybe `add_decoded_pixels` as well? To make it clearer? Otherwise, when reading the code in one year, it's not fully clear that "add_pixels" has any relation with decoded pixels; might be understood as "add pixels to the image" or similar (i.e. not just counting, but doing something with the pixels)

Reply:
`get_pixels_decoded` - sounds like the number of pixels decoded
`get_decoded_pixels` sounds like the getting the actual pixels that have been decoded

Reply:
ack on the `get_pixels_decoded`... 
Any opinion on the `add_pixels`? Maybe even just add the count without even the `add_pixels` method? Somehow add_pixels throws me off the scent when reading the code :thinking:

Reply:
Yeah, we dont really need a public function for this. will remove